src/Accelerometers/Accelerometers.cpp

```diff
@@ -515,7 +513,7 @@ GCodeResult Accelerometers::StartAccelerometer(GCodeBuffer& gb, const StringRef&
 			accelerometerFile->Close();
 			accelerometerFile = nullptr;
 			MassStorage::Delete(accelerometerFileName.c_str(), false);
-			reprap.GetExpansion().AddAccelerometerRun(device.boardAddress, 0);
+			reprap.GetExpansion().AddAccelerometerRun(device.boardAddress, 0, {0});
```

Comment:
Need to pass an array of the correct size here because the compiler doesn't know how many values are expected. Alternatively, add another overload of AddAccelerometerRun that doesn't take the averages as a parameter and instead sets them to zero, since this appears to be done in several places.